sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_autoinstrumentation/configurator.py

```diff
@@ -16,25 +15,35 @@
 )
 from azure.monitor.opentelemetry._diagnostics.diagnostic_logging import (
     AzureDiagnosticLogging,
+    _ATTACH_FAILURE_CONFIGURATOR,
+    _ATTACH_SUCCESS_CONFIGURATOR,
+)
+from azure.monitor.opentelemetry._diagnostics.status_logger import (
+    AzureStatusLogger,
 )
-
-_logger = logging.getLogger(__name__)
 
 
 class AzureMonitorConfigurator(_OTelSDKConfigurator):
     def _configure(self, **kwargs):
         if not _is_attach_enabled():
             warn(_PREVIEW_ENTRY_POINT_WARNING)
         try:
-            AzureDiagnosticLogging.enable(_logger)
+            AzureStatusLogger.log_status(False, "Configurator being configured.")
             super()._configure(**kwargs)
+            AzureStatusLogger.log_status(True)
+            AzureDiagnosticLogging.info(
+                "Azure Monitor Configurator configured successfully.",
+                _ATTACH_SUCCESS_CONFIGURATOR
+            )
         except ValueError as e:
-            _logger.error(
-                "Azure Monitor Configurator failed during configuration due to a ValueError: %s", e
+            AzureDiagnosticLogging.error(
+                "Azure Monitor Configurator failed during configuration due to a ValueError: %s" % str(e),
```

Comment:
Don't think we need to mention the fact that it's a ValueError

Reply:
Then I'll just combine to have one except

Reply:
Done.